scripts/src/main/resources/scripts/command/az

```diff
@@ -1,106 +1,77 @@
 #!/usr/bin/env bash
-
-# autocompletion list
-if [ "${1}" = "shortlist" ]
-then
-  if [ -z "${2}" ]
-  then
-    echo "setup help"
-  fi
-  exit
-fi
-
-if [ -n "${DEVON_IDE_TRACE}" ]; then set -vx; fi
 # shellcheck source=scripts/functions
 source "$(dirname "${0}")"/../functions
 
 function doSetup() {
-  if command -v az &> /dev/null
+  setLocation
+  if [ "${1}" != "silent" ] || [ ! -f "${AZ_FILE}" ] 
   then
-    if [ "${1}" != "silent" ] && ! doIsQuiet
+    if [ -d "${PYTHON_HOME}" ] 
     then
-      doEcho "Azure CLI is already installed at $(command -v az)"
+      doDevonCommand python pip install azure-cli
+      setConfig
+    else
+      doDevonCommand python setup silent
+      doDevonCommand python pip install azure-cli
+      setConfig
```

Comment:
so your IF should now simply check if `az` is already installed locally in my IDE.
If yes there is nothing to install. Otherwise you should always do the installation in one way.
Please note that `doDevonCommand python pip install azure-cli` should always implicitly do `devon python setup silent` for you so you do not need to call that explicitly and you have no need for these two variants here before and after `else`.